pallets/ah-migrator/src/sovereign_account_translation.rs

```diff
@@ -0,0 +1,267 @@
+// Manuall stuff
+use hex_literal::hex;
+use sp_runtime::AccountId32;
+
+// AUTOGENERATED BELOW
+// RC snap path: ../../paseo.snap
+/// List of RC para to AH sibl sovereign account translation sorted by RC account.
+pub const SOV_TRANSLATIONS: &[(AccountId32, AccountId32)] = &[
```

Comment:
@seadanda we maybe want to re-generate this for Paseo, although i might just merge all networks into one (Paseo, Polkadot & Kusama).  
Just for now its simpler like this to keep it per network.